packages/fx-core/src/component/utils/common.ts

```diff
@@ -55,3 +55,21 @@ export function asFactory<T>(keyValidators: KeyValidators<T>) {
     throw PrerequisiteError.somethingIllegal("Deploy", "data", "plugins.bot.InvalidData");
   };
 }
+
+// Expand environment variables in content. The format of referencing environment variable is: ${{ENV_NAME}}
+export function expandEnvironmentVariable(content: string): string {
+  const placeholderRegex = /\${{[a-zA-Z_][a-zA-Z0-9_]*}}/g;
```

Comment:
I think environment variable parameters should not be allowed to start with an underscore

Reply:
Based on following answer, env name can start with underscore (only digit is not allowed as first character):
https://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names